### PR TITLE
#3949 Pipeline page element property view (render with settings)

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3949)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/configs/cypress.e2e.json [hunk 1]
```diff
@@ -11,6 +11,6 @@
   "requestTimeout": 30000,
   "defaultCommandTimeout": 60000,
   "watchForFileChanges": false,
-  "viewportWidth": 1280,
+  "viewportWidth": 1600,
```
Comment (saivirtue) [inline ohara-manager/client/cypress/configs/cypress.e2e.json]: 😄

**Comment by konekoya** [comment]
@wu87988622 the conflict has been resolved

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
@chia7712 All tests passed. The PR is ready to go :)

**Pull request merged**